src/sentry/incidents/logic.py

```diff
@@ -590,19 +590,17 @@ def create_alert_rule(
                 for project in excluded_projects
             ]
             AlertRuleExcludedProjects.objects.bulk_create(exclusions)
-        elif monitor_type == AlertRuleMonitorType.ACTIVATED and projects:
-            # initialize projects join table for activated alert rules
-            arps = [
-                AlertRuleProjects(alert_rule=alert_rule, project=project) for project in projects
-            ]
-            AlertRuleProjects.objects.bulk_create(arps)
 
         if monitor_type == AlertRuleMonitorType.ACTIVATED and activation_condition:
             # NOTE: if monitor_type is activated, activation_condition is required
             AlertRuleActivationCondition.objects.create(
                 alert_rule=alert_rule, condition_type=activation_condition.value
             )
 
+        # initialize projects join table for alert rule
```

Comment:
TODO: split this into a separate PR

Reply:
still want to pull t his into another pr?

Reply:
pulled into https://github.com/getsentry/sentry/pull/68022